src/app/component/general/eco-news/filter-news/filter-news.component.ts

```diff
@@ -0,0 +1,61 @@
+import { Component, OnInit, Output, EventEmitter } from '@angular/core';
+
+@Component({
+  selector: 'app-filter-news',
+  templateUrl: './filter-news.component.html',
+  styleUrls: ['./filter-news.component.css']
+})
+
+export class FilterNewsComponent implements OnInit {
+
+  private filters = [{ name: 'news', isActive: false },
```

Comment:
bad formatting, please replace the object to the new line

Reply:
Fixed